src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/PaperCaseCreateContestedAboutToSubmitHandlerTest.java

```diff
@@ -139,4 +149,14 @@ void testUpdateApplicantInRefugeTabCalled() {
             mockedStatic.verify(() -> RefugeWrapperUtils.updateApplicantInRefugeTab(caseDetails), times(1));
         }
     }
+
+    @Test
+    void testSetFinancialRemediesCourtDetailsCalled() {
+        FinremCallbackRequest callbackRequest = buildFinremCallbackRequest(CONTESTED_VALIDATE_HEARING_SUCCESSFULLY_JSON);
+
+        handler.handle(callbackRequest, AUTH_TOKEN);
+        verify(finremCaseDetailsMapper, times(1)).mapToCaseDetails(any(FinremCaseDetails.class));
```

Comment:
I don't think you need to verify that the mapper was called. This is an internal implementation detail. What's the value in verifying it was called? If someone refactors the code so that we don't need to map from one representation to the other then your test fails - even though the functionality is correct.